lib/runner.js

```diff
@@ -146,7 +146,9 @@ function runner(script, payload, options) {
   //
   let runnerPlugins = [];
   _.each(runnableScript.config.plugins, function tryToLoadPlugin(pluginConfig, pluginName) {
-    let requireString = 'artillery-plugin-' + pluginName;
+    let pluginPackage = pluginConfig.package || runnableScript.config.pluginsPackage;
+    let pluginPrefix = pluginPackage ? pluginPackage + '/' : 'artillery-plugin-';
```

Comment:
I think this should be simplified and just allow for the full package name to be specified, e.g. `@myorg/my-private-plugin`. Otherwise the logic for what gets loaded isn't consistent:

1. If `package` is specified, it becomes value of that + plugin name as specified in the `config` section of the script. "Package" on its own is somewhat misleading as well, as it would imply the full package name rather than the package scope as the code expects it to be.
2. Otherwise it's `artillery-plugin-` + plugin name.